data/presets/man_made/charge_point.json

```diff
@@ -0,0 +1,33 @@
+{
+    "icon": "fas-charging-station",
+    "fields": [
+        "brand",
+        "operator",
+        "capacity",
+        "access_simple",
+        "fee",
+        "payment_multi_fee",
+        "charge_fee",
+        "ref"
+    ],
```

Comment:
I'm not so sure about the fields to be used for this preset. The [wiki page](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dcharge_point) (and [proposal](https://wiki.openstreetmap.org/wiki/Proposal:EV_Charging_Station_Mapping)) don't seem to suggest to use the tags from the main `amenity=charging_station` like `brand`, `operator`, `fee` also on the individual charging points of the station. Isn't that similar to how the benches inside a park typically don't repeat the operator of the park, or the individual `parking_space`s of a car park don't repeat the `fee` the of the parking area they are in, for example?